seed/static/seed/js/services/uploader_service.js

```diff
@@ -44,6 +44,23 @@ angular.module('BE.seed.service.uploader', []).factory('uploader_service', [
       });
     };
 
+    // TODO: Temporary dropping point for the actual PM functions
```

Comment:
This is another temporary drop for these functions, we should discuss where these will eventually live.